pom.xml

```
@@ -95,6 +96,11 @@
             <artifactId>jackson2-api</artifactId>
             <version>${jackson2-api.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>bouncycastle-api</artifactId>
+            <version>${bouncycastle-api.version}</version>
```

```suggestion
            <version>2.17</version>
```

…like this. We generally use POM properties only to avoid repetition.

---

Done!